src/types/payloadType.ts

```diff
@@ -1,7 +1,7 @@
 export type JwtPayloadType = {
-    id: number,
-    firstName: string,
-    lastName: string,
-    displayName: string,
-    role: string,
-}
\ No newline at end of file
+  id: number;
+  firstName: string;
+  lastName: string;
+  displayName: string;
+  role: string;
+};
```

Comment:
Members of type ends with , not with ; 
and last member role: string doesnt need to be ended with ,
export type JwtPayloadType = {
    id: number,
    firstName: string,
    lastName: string,
    displayName: string,
    role: string
};

Reply:
Maybe... ;)